docs/CONTRIBUTING.md

```diff
@@ -6,8 +6,8 @@ Releases are made on a reasonably regular basis by the maintainers (HashiCorp st
 
 Release process:
 
- 1. Update [`version/VERSION`](https://github.com/hashicorp/hc-install/blob/main/version/VERSION) to remove `-dev` suffix and set it to the intended version to be released
- 1. Wait for [`build` workflow](https://github.com/hashicorp/hc-install/actions/workflows/build.yml) to finish
- 1. Run the Release workflow with the appropriate version (matching the one in `version/VERSION`) & SHA (long one).
- 1. Wait for a message in the Slack channel saying that authorisation is needed to promote artifacts to production. Click on the link and approve.
- 1. Once notified that promotion is successful, go to https://github.com/hashicorp/crt-workflows-common/actions/workflows/promote-production-packaging.yml, locate the hc-install promote-production-packaging workflow, and approve.
+1. Update [`version/VERSION`](https://github.com/hashicorp/hc-install/blob/main/version/VERSION) to remove `-dev` suffix and set it to the intended version to be released
+2. Wait for [`build` workflow](https://github.com/hashicorp/hc-install/actions/workflows/build.yml) to finish
+3. Run the Release workflow with the appropriate version (matching the one in `version/VERSION`) & SHA (long one).
+4. Wait for a message in the Slack channel saying that authorisation is needed to promote artifacts to production. Click on the link and approve.
+5. Once notified that promotion is successful, go to <https://github.com/hashicorp/crt-workflows-common/actions/workflows/promote-production-packaging.yml>, locate the hc-install promote-production-packaging workflow, and approve.
```

Comment:
I don't mind the whitespace removal but I would strongly prefer the indexes to stay as-is, i.e. `1`. The reason is that if a human goes to add another item into the middle of the list, they have to re-index any following numbers.

Also similar question here about the `<URL>` syntax.

Reply:
Sure thing, I understand that logic